### DISCUSSION TITLE
Welcome to Digital-IDE Discussions!

### DISCUSSION THREAD

**Discussion by LSTM-Kirigaya** [Digital-EDA/Digital-IDE#56] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlxRubel** [comment]
Is there a way to disable the display of "input/output" hint in module instances?
<img width="430" height="95" alt="image" src="https://github.com/user-attachments/assets/6c76174b-722c-48ee-bcfc-c86cd32b6f4d" />
